spec/rails_helper.rb

```diff
@@ -7,6 +7,7 @@
 # Prevent database truncation if the environment is production
 abort("The Rails environment is running in production mode!") if Rails.env.production?
 require 'rspec/rails'
+require 'support/factory_bot'
```

Comment:
we added the code from support/factory_bot directly into the rails helper. I think your way is better practice in general, although it felt like overkill for a project of this size. We'll just have to make sure we're not doing both when we merge the branches

Reply:
I did add the code to rails helper the first time I used it, I think I defaulted to the setup guide. Whichever way is fine.